docs/reference/migration/migrate_7_0/sql.asciidoc

```diff
@@ -0,0 +1,9 @@
+[[breaking_70_sql_changes]]
+=== SQL plugin changes
+
+==== Grouping by columns with missing values will create an additional group 
+
+An additional group will be present in the result of requests containing a
```

Comment:
If this goes to 6.x then this shouldn't actually be committed to `migrate_7_0`, only the 6.x version.

Reply:
Hm... good point @nik9000... a breaking change happens once on the versions timeline.

Reply:
This breaking change in 6.x should go to `\docs\reference\migration\migrate_6_5.asciidoc`, right?